README.md

```diff
@@ -211,9 +198,9 @@ Example:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- [ ] **[use API]**
+<!-- - [ ] **[new_feature_2]** -->
+<!-- - [ ] **[new_feature_3]** -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [**Optiona**l] Hi @Reggeditt, the section **FAQ** is an optional. And you seem to have no question and answer for the moment. What do you think of getting rid of it? This will render our README file more descriptive :+1:

Reply:
I'll get right on it. thanks for the suggestion.